packages/tools/create-kadena-app/templates/angular/src/app/app.component.html

```diff
@@ -0,0 +1,36 @@
+<div>
+ <main class="grid">
+   <div class="card">
+     <div class="header">
+       <h1 class="title">Welcome to <span>Kadena!</span></h1>
+     </div>
+     <p>This is the Kadena starter template using nextjs to help you get started on your blockchain development. Use the form below
+       to interact with the Kadena blockchain using <code>@kadena/client</code> and edit <code>src/app/app.component.html</code> to get started.</p>
+   </div>
+   <div class="card">
+     <h3 class="cardTitle">Interact with the blockchain</h3>
+     <section class="cardSection">
+       <h4>My Account</h4>
+       <input id="account" [(ngModel)]="account" placeholder="Please enter a valid k:account" />
```

Comment:
Same as the comment below about `ngModel` bi-directional binding!

Reply:
See the remark on the previous comment